pkg/cmd/roachtest/tests/restore.go

```diff
@@ -296,6 +296,15 @@ func registerRestore(r registry.Registry) {
 			backup:   makeBackupSpecs(backupSpecs{}),
 			timeout:  1 * time.Hour,
 		},
+		{
+			// Benchmarks if per node throughput remains constant if the cluster
+			// is multi-region.
+			hardware: makeHardwareSpecs(hardwareSpecs{
+				nodes: 6,
+				zones: []string{"us-east-1b", "us-east-2b"}}), // These zones are AWS-specific.
```

Comment:
this would make a cluster across different availability zones, not necessarily different regions (us-west...) . I think we want to use different regions?

I'm not entirely confident that a "2 region, three nodes in each region" is a representative simple MR topology, mostly bc i don't now much about mr topologies. Perhaps bring it up in discussion or demo to ask the team?